lib/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "behave-graph",
+  "version": "0.9.9",
+  "source": "src/index.ts",
+  "main": "dist/behave-graph.cjs.js",
+  "module": "dist/behave-graph.esm.js",
+  "types": "dist/behave-graph.cjs.d.ts",
+  "sideEffects": false,
+  "preconstruct": {
+    "entrypoints": [
+      "index.ts",
+      "exec-graph.ts",
```

Comment:
2 new entrypoints so that these scripts are built, and we can call them directly